db/seeds.rb

```diff
@@ -5,3 +5,17 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+holliday = Hotel.create!(name: 'Holliday Inn', location: 'Buxton')
+harbor = Hotel.create!(name: 'Portland Harbor', location: 'Portland') 
+room_1 = harbor.rooms.create!(rate: 180, suite: "Presidential")
+room_2 = harbor.rooms.create!(rate: 185, suite: "Vice-Presidential")
+room_3 = holliday.rooms.create!(rate: 160, suite: "Honeymoon")
+room_4 = holliday.rooms.create!(rate: 10, suite: "Broom Closet")
+steve = Guest.create!(name: 'Steve', nights: 3)
+dave = Guest.create!(name: 'Dave', nights: 5) 
+jim = Guest.create!(name: 'Jim', nights: 1)
+GuestRoom.create!(guest: steve, room: room_1)
+GuestRoom.create!(guest: dave, room: room_1)
+GuestRoom.create!(guest: steve, room: room_2)
+GuestRoom.create!(guest: dave, room: room_3)
+GuestRoom.create!(guest: dave, room: room_4)
```

Comment:
When you're working through and IC, it really isn't required to have to create any seed data since you will be working in the test files for the most part. If you wanted to see what the page looked like in the browser, you would just use `save_and_open_page`